__init__.py

```diff
@@ -278,6 +303,15 @@ def start_recording(self, context) -> bool:
                 f'-i ":{addon_prefs.audio_device_darwin}" '
                 f"-framerate {framerate} {self.filepath}"
             )
+        elif os_platform == 'Windows':
+            filepath = chr(34)+(self.filepath)+chr(34)#.replace(chr(92), chr(92)+chr(92))+chr(34)
```

Comment:
Make sure you run `black` for code formatting.

Reply:
This and the other comment has been fixed in the updated patch, however the conflict found by github looks absurd to me?